featurebyte/query_graph/model/graph.py

```diff
@@ -54,11 +54,16 @@ class QueryGraphModel(FeatureByteBaseModel):
     _node_topological_order_map: Dict[str, Any] = PrivateAttr(default_factory=dict)
 
     def __repr__(self) -> str:
-        return self.json(by_alias=True, indent=4)
+        return self.model_dump_json(by_alias=True, indent=4)
 
     def __str__(self) -> str:
         return repr(self)
 
+    def __eq__(self, other: Any) -> bool:
+        if isinstance(other, QueryGraphModel):
+            return bool(self.dict() == other.dict())
+        return False
```

Comment:
Curious when will we do equality comparison between two query graph models?

Reply:
We mainly use the comparison in our tests.